unit/e2e_tests/ruby/spec/models/check_deposit_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+require 'json'
+require 'date'
+
+# Unit tests for OpenapiClient::CheckDeposit
+# Automatically generated by openapi-generator (https://openapi-generator.tech)
+# Please update as you see appropriate
+RSpec.describe 'CheckDeposit' do
+  before do
+    configure_tests
+  end
+
+  describe 'test an instance of CheckDeposit' do
+    let(:api_instance) { OpenapiClient::CreateCheckDepositApi.new(OpenapiClient::ApiClient.new(configuration)) }
+
+    it 'should create a checkDeposit' do
+      request = { data: OpenapiClient::CreateCheckDeposit.new(type: 'checkDeposit',
+                                                              attributes:
+                                                                OpenapiClient::CreateCheckDepositAttributes.new(
```

Comment:
I don't understand why relationships is converted to hash, but attributes not? is it needed only for relationships? or both, or none?

Reply:
@nirmaoz you are right, for the relationship it is not needed